rust/libnewsboat/src/keycombination.rs

```diff
@@ -80,7 +79,7 @@ impl KeyCombination {
 }
 
 fn alphabetic(input: &str) -> IResult<&str, char> {
-    verify(anychar, |c: &char| is_alphabetic(*c as u8))(input)
+    verify(anychar, |&c| ((c as u32) < 256) && c.is_alpha()).parse(input)
```

Comment:
To me, this would have been clearer:

```suggestion
    verify(anychar, |&c| c.is_ascii() && c.is_alpha()).parse(input)
```

Reply:
Nice!
Suggestion applied via amended commit